aiida_pseudo/cli/family.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+"""Commands to inspect or modify the contents of pseudo potential families."""
+import json
+
+import click
+
+from aiida.cmdline.utils import decorators, echo
+
+from ..groups.mixins import RecommendedCutoffMixin
+from .params import arguments, options
+from .root import cmd_root
+
+
+@cmd_root.group('family')
+def cmd_family():
+    """Command group to inspect or modify the contents of pseudo potential families."""
+
+
+@cmd_family.group('cutoffs')
+def cmd_family_cutoffs():
+    """Command group to inspect or modify the recommended cutoffs of pseudo potential families."""
+
+
+@cmd_family_cutoffs.command('set')
+@arguments.PSEUDO_POTENTIAL_FAMILY()
+@click.argument('cutoffs', type=click.File(mode='rb'))
+@options.STRINGENCY(required=True)
+@decorators.with_dbenv()
+def cmd_family_cutoffs_set(family, cutoffs, stringency):
+    r"""Set the recommended cutoffs for a pseudo potential family.
```

Comment:
The raw format messes up your beautiful formatting 😭 

```suggestion
    """Set the recommended cutoffs for a pseudo potential family.
```

Reply:
Weird, I could have sworn that I added this because without it the `\b` marker didn't seem to be respected. But indeed, removing it keeps the formatting as intended

Reply:
Ah I see why, it is `pydocstyle` that complains about the `\b` marker saying I should use `r` when using backslashes. Will try to add an exception